packages/nextjs/components/Header.tsx

```diff
@@ -58,14 +58,15 @@ export const Header = () => {
     <div className="sticky lg:static top-0 navbar bg-base-100 min-h-0 flex-shrink-0 justify-between z-20 shadow-md shadow-secondary">
       <div className="navbar-start w-auto lg:w-1/2">
         <div className="lg:hidden dropdown" ref={burgerMenuRef}>
-          <button
```

Comment:
Do we also need to change it here?

It seems that we are not using the CSS magic from DaisyUI here, right? (tabIndex stuff) 

Here it looks like we are relying on react state (isDrawerOpen).

I personally prefer using React state (instead of CSS magic), but we are using it in some places (e.g. for modals). It feels comfy, because you don't have to manage any state.... but is it maybe a bit hacky? Maybe it's just me haha

What do you think, Shiv?

Reply:
Just went through the code and it seems that we are just maintaining the state `isDrawerOpen ` to close dropdown list through `Hamburger icon` thats it. 

https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/ef58cb5d-9899-48cd-b6cc-8535ce993366



Rest all is handled by daisyUI like showing dropdown list in the `absolute` position, small animations and all, we used the `dropdown` class in the parent `div` so `tabIndex` was necessary to solve problem mentioned in the first video of the main comment. 

> I personally prefer using React state (instead of CSS magic),... but is it maybe a bit hacky?

Lol, even I have the same feeling! Should we go this route? I think we need to remove `dropdown` class and its all related classNames and add `absolute`  positioning and other classes ourselves not a big deal I think, also we will be more consistent on our source of truth for handling `opening`/ `closing` only depend on `state` unlike in current situation it is handled by both `css` and `js`, just wanted to confirm before I start making changes 🙌

Reply:
Let's keep like this for now. But it's something that we might want to do in the future.